assets/src/features/room-page/utils/usePinning.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+
+export type PinningApi = {
+    pinnedTileId: string;
+    pin: (tileId: string) => void;
+    unpin: () => void;
+} 
+
+const usePinning = () : PinningApi => {
+    const [pinnedTileId, setPinnedTileId] = useState<string>("");
```

Comment:
I think null would be better here instead of an empty string. It would be the correct ID or null. It would be easier to handle